cmake/find/Findbenchmark.cmake

```diff
@@ -29,7 +29,7 @@ if(BENCHMARK_INCLUDE_DIR AND BENCHMARK_LIBRARY)
   add_library("benchmark::library" UNKNOWN IMPORTED)
   set_target_properties("benchmark::library" PROPERTIES
       INTERFACE_INCLUDE_DIRECTORIES "${BENCHMARK_INCLUDE_DIR}"
-			INTERFACE_COMPILE_OPTIONS -fno-omit-frame-pointer
+      INTERFACE_COMPILE_OPTIONS -fno-omit-frame-pointer
```

Comment:
Probably a bit [more ](https://msdn.microsoft.com/en-us/library/2kxx5t2c.aspx) logic required for the MSVS generator: